docs/resources/synciq_replication_job.md

```diff
@@ -19,22 +19,59 @@ linkTitle: "powerscale_synciq_replication_job"
 page_title: "powerscale_synciq_replication_job Resource - terraform-provider-powerscale"
 subcategory: ""
 description: |-
-  Resource for managing SyncIQReplicationJobResource on OpenManage Enterprise.
+  Resource for managing SyncIQ ReplicationJob on PowerScale. This resource can be used to manually trigger the replication job to replicate data from source powerscale cluster to a target powerscale cluster.
```

Comment:
Can we mention here that it will only trigger but not monitor or something like that